app/controllers/application_controller.rb

```diff
@@ -45,6 +46,16 @@ def check_team_authorization(team_id: params[:team_id])
     redirect_to root_path, status: :forbidden
   end
 
+  def check_member_authorization(user_id: params[:user_id])
+    if user_id.present?
```

Comment:
```suggestion
    return unless user_id.present?
    
```
this way we don't need to inded and do end at the end ;)

Reply:
ah yeah that is better :) have amended.